src/css/custom.css

```diff
@@ -126,7 +126,7 @@
   text-align: center;
 }
 
-.text-slate\/60 {
+.text-email\/60 {
```

Comment:
nit: let's drop the "\/60" part too

(My guess is that "slate/60" was a color name from some color system that we're not really using.)